actions/v2/internal/integrationtests/testabilities/ability_integration_tests.go

```diff
@@ -4,14 +4,51 @@ import (
 	"testing"
 
 	"github.com/bitcoin-sv/spv-wallet/actions/testabilities"
+	"github.com/bitcoin-sv/spv-wallet/engine/testabilities/testmode"
 )
 
+// DBOption represents a database configuration option for integration tests
+type DBOption func(t testing.TB)
+
+// WithoutCleanup tells the test not to clean up the database after running
+func WithoutCleanup() DBOption {
+	return func(t testing.TB) {
+		testmode.WithoutCleanup(t)
+	}
+}
+
+// New creates a new integration test fixture with PostgreSQL container by default
 func New(t testing.TB) (given IntegrationTestFixtures, when IntegrationTestAction, then IntegrationTestAssertion) {
+	setPostgresContainer(t)
```

Comment:
Does it mean, that it will run in integration tests by default?

I was rather thinking about having integration tests on both data bases, not only on postgres